model-engine/model_engine_server/inference/forwarding/http_forwarder.py

```diff
@@ -162,6 +162,7 @@ def entrypoint():
         str(args.num_workers),
         *envs,
         "model_engine_server.inference.forwarding.http_forwarder:app",
+        *extra_args,
```

Comment:
oh wait hmm I think it might be good to just hardcode in the termination grace period seconds as a default here idk

Reply:
`extra_args` feels like it should be a dict to me (e.g. `--key1 val1 --key2 val2` from `{key1: val1, key2: val2}`), is there such thing as `extra_kwargs`? 

this way we can set a default of graceful-timeout and override it if necessary

Reply:
yeah it's fine to pass; I was hoping for a dict but if we don't get that from argparse then we can just directly pass it